test_upload_form.py

```diff
@@ -57,3 +57,90 @@ def test_upload_file(filename='benoitcommercon.csv',
     up_sc = github_helpers.close_pull_request('uploads', uppull)
 
     return S,r1,r2,r3
+
+def test_upload_rathborne(filename='rathborne2009_table2.ecsv',
```

Comment:
Should we include this dataset (or a portion) for use in these unit tests? Same with 'benoitcommercon.csv'.